docs/ci.md

```diff
@@ -0,0 +1,38 @@
+LinuxKit CI
+===========
+
+The LiunxKit CI system is uses [DatakitCI](https://github.com/moby/datakit/tree/master/ci)
+The configuration can be found [here](https://github.com/linuxkit/linuxkit-ci)
+
+## PR Testing
+
+Each PR is tested on disposable VM's spawned in Google Cloud Platform
+This machine has no privileges or credentials to talk to GCP or other cloud platforms.
+
+FIXME: Add instructions on how to build a base image for LinuxKitCI and GCP
```

Comment:
@talex5 I remember we discussed this briefly but don't recall reaching a conclusion.
Using `moby` to build the CI images would be awesome.